bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java

```diff
@@ -586,17 +489,27 @@ private void handleMediaControlCommand(ChannelUID channelUID, Command command)
         }
     }
 
-    private void seekToPercentage(int percentage) throws SyncCallback.SyncCallbackError, ApiClientException {
+    private void seekToPercentage(Command command)
+            throws IllegalArgumentException, SyncCallback.SyncCallbackError, ApiClientException {
```

Comment:
You added `IllegalArgumentException`.
What is the code that can trigger this exception ?

Reply:
Ummm, yes maybe it wasn't the best decision.

The method could throw a NumberFormatException which extends from IllegalArgumentException.
As the parseItemUUID could also throws an IllegalArgumentException, I simplified it just adding common catch for the IllegalArgumentException to handle all as the same in the handleCommand method. But it's probably most accurate to have a "throws NumberFormatException" in both two methods.

Reply:
I have changed it to NumberFormatException in both methods.

Reply:
I have added a specific catch for NumberFormatException.